src/interpreter/index.ts

```diff
@@ -1526,10 +1533,16 @@ export class Interpreter implements Expr.Visitor<BrsType>, Stmt.Visitor<BrsType>
     }
 
     evaluate(this: Interpreter, expression: Expr.Expression): BrsType {
+        try {
```

Comment:
Why do we need to surround this with `try-catch`? Isn't `location` required on the interface for `Statement` and `Expression`?

Reply:
This is admittedly because of my laziness — a lot of unit tests for the `interpreter/` tree use raw AST nodes as input, but attempting to access `location` winds up throwing an error 🤢 

I'll see if there's a way to fix that without blowing up the diff!

Reply:
Fixed!